/cone_detection:
  ros__parameters:
    sub_topic: "/ZOE3/os_node/points"
    pub_topic: "/ZOE3/os_node/cone_points"
    pub_cloud_frame: "ZOE3/os_sensor"
    leaf_size_x: 0.2
    leaf_size_y: 0.2
    leaf_size_z: 0.2
    downsample_all_data: true

